### DISCUSSION TITLE
{{{commitMessagePrefix}}}  does not match documented format

### DISCUSSION THREAD

**Discussion by tux-84** [renovatebot/renovate#23508] [Request Help] [answered]
### Describe the proposed change(s).

The current format stated in the documentation is:
```
"{{{commitMessagePrefix}}} {{{commitMessageAction}}} {{{commitMessageTopic}}} {{{commitMessageExtra}}} {{{commitMessageSuffix}}}"
```

However, the real commit message has a `:` right after the prefix. Our push rules require a prefix and then a space. This results in a push rule validation error.

### Describe why we need/want these change(s).

RENOVATE_COMMIT_MESSAGE_PREFIX=foo should not result in `foo:` but `foo`.

**Comment by HonkingGoose** [comment]
The [`commitMessage` docs (deprecated)](https://docs.renovatebot.com/configuration-options/#commitmessage) says the `default` is:

```
"{{{commitMessagePrefix}}} {{{commitMessageAction}}} {{{commitMessageTopic}}} {{{commitMessageExtra}}} {{{commitMessageSuffix}}}"
```

So it's _not_ showing the `:` character.

---

The [`commitMessagePrefix` docs](https://docs.renovatebot.com/configuration-options/#commitmessageprefix) also does _not_ mention the `:` character.

---

Many users probably rely on the current behavior, so we should probably update the docs? I don't know what's the best place to put the `:` character info. @rarkins do you have any ideas?

**Reply by rarkins** [reply, marked as answer]
Yes, I think the only option is to document it. Today our code always appends a `:` if none exists

**Reply by HonkingGoose** [reply]
I'm resolving this discussion with PR:

- #23611
